src/System.Management.Automation/utils/tracing/PSSysLogProvider.cs

```diff
@@ -17,8 +17,11 @@ internal class PSSysLogProvider : LogProvider
     {
         private static SysLogProvider s_provider;
 
-        // by default, do not include analytic events
-        internal const PSKeyword DefaultKeywords = (PSKeyword) (0xFFFFFFFFFFFFFFFF & ~(ulong)PSKeyword.UseAlwaysAnalytic);
+        // by default, do not include channel bits
+        internal const PSKeyword DefaultKeywords = (PSKeyword) (0x00FFFFFFFFFFFFFF);
```

Comment:
The default keyword should be `0x0FFFFFFFFFFFFFFF`. The diff bit for `UseAlwaysOperational` and `UseAlwaysAnalytic` is only in the first 4 bits.
```
UseAlwaysOperational = 0x8000000000000000,
UseAlwaysAnalytic    = 0x4000000000000000,
```

Reply:
That's as intended; channel bits use the high order byte.

Reply:
Got it. Closed.